tests/rust/test_private_and_public_key.py

```diff
@@ -0,0 +1,45 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+
+import pytest
+
+
+@pytest.mark.rust
+def test_private_key():
+    from parsec.crypto import (
+        _PyPrivateKey,
+        _RsPrivateKey,
+        PrivateKey,
+        PublicKey,
+        _PyPublicKey,
+        _RsPublicKey,
+    )
+
+    assert PrivateKey is _RsPrivateKey
+    assert PublicKey is _RsPublicKey
+
+    KEY = b"a" * 32
+    MESSAGE = b"test private key"
+
+    rs_pvk = _RsPrivateKey(KEY)
+    py_pvk = _PyPrivateKey(KEY)
+
+    rs_pbk = rs_pvk.public_key
+    py_pbk = py_pvk.public_key
+
+    assert isinstance(rs_pvk, _RsPrivateKey)
+    assert isinstance(py_pvk, _PyPrivateKey)
+
+    assert isinstance(rs_pbk, _RsPublicKey)
+    assert isinstance(py_pbk, _PyPublicKey)
+
+    assert repr(rs_pvk) == repr(py_pvk)
+
```

Comment:
Maybe add a test to compare two PrivateKey, like `PrivateKey(b"a" * 32) == PrivateKey(b"a" * 32)` and `PrivateKey(b"a" * 32") != PrivateKey(b"b" * 32)` just to check the `richcmp`.